.travis.yml

```diff
@@ -21,20 +24,18 @@ cache:
   - packages
 
 addons:
-  postgresql: 9.6
   apt:
     sources:
       - sourceline: 'ppa:marutter/c2d4u'
       - sourceline: 'ppa:nschloe/hdf5-backports'
+      - sourceline: 'ppa:ubuntugis/ppa' # for GDAL 2 binaries
```

Comment:
Why not also install libgdal-dev, which is what provides the right libraries to compile rgdal.

Reply:
We're installing it already (line 49) -- it was just finding[libgdal-dev 1.10 from trusty/universe](https://travis-ci.org/PecanProject/pecan/builds/396665460#L1083) before, and after adding this line it now finds [libgdal-dev 2.1 from ubuntugis](https://travis-ci.org/PecanProject/pecan/builds/396861337#L1110) without compiling.

Reply:
I think you might be able to install `libgdal1-dev`

Reply:
At some point I tried that with apt binaries and found that it always removed the first-installed version when I tried to the other, but I don't think I revisited it with sudo enabled. Would you be willing to take a stab at it?